docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -398,3 +398,56 @@ To update the schema, and then reindex the content, use the following commands:
 php bin/console ibexa:elasticsearch:put-index-template --overwrite
 php bin/console ibexa:reindex
 ```
+
+## v4.6.2
+
+#### Database update
+
+Run the following scripts:
+
+=== "MySQL"
+
+    ``` sql
+    mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.6.1-to-4.6.2.sql
+    ```
+
+=== "PostgreSQL"
+
+    ``` sql
+    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.6.1-to-4.6.2.sql
+    ```
+
+## v4.6.3
+
+### Notification config update
+
+The configuration of the package `ibexa/notifications` has changed.
+This package is required by other packages, such as `ibexa/connector-actito` for [Transactional emails](https://doc.ibexa.co/en/latest/commerce/transactional_emails/transactional_emails/), `ibexa/payment`, or `ibexa/user`.
+
+If you are already customizing the package `ibexa/notifications` configuration, and used SiteAccess aware configuration to change `Notification` subscriptions, you have to manually change your configuration by using new node name `notifier` instead of the old `notifications`.
+
+For example, the following 4.6.2 config:
+
+```yaml
+ibexa:
+    system:
+        my_siteacces_name:
+            notifications: # deprecated
```

Comment:
```suggestion
            notifications: # old
```
Nitpick: `deprecated` for me means "no longer recommended, but still working - will be removed in the future".

As far I understand this change needs to be done now, so I'd avoid this word.

Reply:
I was thinking "deprecated in 4.6.2" but this isn't true as we didn't know it will be renamed.